contrib/http_util/src/main/java/io/opencensus/contrib/http/HttpServerHandler.java

```diff
@@ -176,11 +177,18 @@ private void recordStats(
     TagContext startCtx =
         tagger
             .toBuilder(context.tagContext)
-            .put(HTTP_SERVER_METHOD, TagValue.create(methodStr == null ? "" : methodStr))
-            .put(HTTP_SERVER_ROUTE, TagValue.create(routeStr == null ? "" : routeStr))
+            .put(
+                HTTP_SERVER_METHOD,
```

Comment:
I would think we wouldn't want to propagate some of these tags but instead have the endpoints set them explicitly during message processing.

Reply:
Changing these tags to non-propagating by default would be a breaking change, though the impact could be minimal since HTTP handlers are just released in v0.19. WDYT? @rghetia

Reply:
as discussed. HTTP handlers were not propagating tags. So no breaking change.